base/compiler/ssair/show.jl

```diff
@@ -26,7 +28,7 @@ end
 print_ssa(io::IO, @nospecialize(val), argnames) = Base.show(io, val)
 
 
-function print_node(io::IO, idx::Int, @nospecialize(stmt), used, argnames, maxsize; color = true, print_typ=true)
+function print_node(io::IO, idx::Int, @nospecialize(stmt), used, argnames, maxsize; color::Bool=true, print_typ::Bool=true)
```

Comment:
Can the `@nospecialize` be removed here since it is in a @nospecialize true block?

Reply:
It could, but I like the reminders that this code will run very slowly if you aren't careful